.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: android
+
+android:
+  components:
+    - tools
+    - platform-tools
+
+jdk:
+  - oraclejdk8
+
+before_script:
+  - mkdir "$ANDROID_HOME/licenses" || true
+  - echo "d56f5187479451eabf01fb78af6dfcb131a6481e" > "$ANDROID_HOME/licenses/android-sdk-license"
+
+before_install:
+  - pip install --user codecov
+
+install: true
+
+script: ./gradlew clean build
```

Comment:
Maybe 'check' instead of 'build'? 🤔

Reply:
Don't have a strong opinion here. Why would you go for check?

Reply:
Because check runs everything, checkstyles, tests and so on.

Reply:
Does not build the final release apk though.

Reply:
Maybe we should have both then?

Reply:
Also I found a solution for accepting all licenses (also future ones) on CI 
`yes | sdkmanager --licenses` 
Do you think it would make sense to add it as well?

Reply:
`check` is running when `build` is executed. Yup that would make sense. How would you add `yes | sdkmanager --licenses`? Like this `yes | sdkmanager --licenses > "$ANDROID_HOME/licenses/android-sdk-license"` ?

Reply:
@vanniktech You don't need to specify anything for `yes | sdkmanager --licenses`

you can just put the line before the build and it will work, it will go through all licenses and accept them. storing this sdk license file will be no longer needed then :)